tests/data/3/gencodedocs.expected.doc_func.md

```diff
@@ -1,6 +1,6 @@
 #### func_a
-`def func_a(a, b=None, **kwargs)`
+`def func_a(a, b, kwargs=None)`
```

Comment:
`def func_a(a, b=None, **kwargs)` is the accurate definition (see https://github.com/20c/pymdgen/pull/21/files#diff-b06ce02e2aac4bb01e762c0a8a54545a01af7cc84b5745511500907c4b0abd12R4) - why are we losing it?

Reply:
@vegu still not sure about that, the tests are failing due to that

Reply:
@vegu fixed, kindly review it again